migrations/6_initialize_721token.js

```diff
@@ -29,19 +23,23 @@ module.exports = async (deployer, network, accounts) => {
 
         case 'rinkeby':
           erc20TokenAddress = '0xb05e292f89c6a82f5ed1be694dc7b6444866b364'
-          initialFees = 10
+          initialFees = 10 ** 18 // 1 token
```

Comment:
This may need to be a BigNumber, otherwise it'll loose some precision.

Reply:
👍